protocol/auditor.go

```diff
@@ -105,7 +106,7 @@ func (a *AudState) checkSTRAgainstVerified(str *DirSTR) error {
 		return CheckBadSTR
 	}
 
-	return nil
```

Comment:
Do you mean only for this function because it's only a helper function, or in all the cases in which I replaced `nil` with `CheckPassed`? I feel like it would definitely make sense to return `CheckPassed` from `AuditDirectory()` at least.